### DISCUSSION TITLE
A topic on cuDNN installation

### DISCUSSION THREAD

**Discussion by michail935** [bmaltais/kohya_ss#438] [Q&A]
I have a question, if anyone knows, tell me. I have an Nvidia 4080 video card. I recently installed the latest version of cuDNN on my PC, literally two days ago.I installed cuDNN in the "CUDE driver".   According to the instructions in the Nvidia drivers on my PC. Today I downloaded and started installing "the Kohya's GUI". I selected "bf16" when installing it. As I understand it is for those who have 30-40 series cards or? Do I need to download more bilibrary with cuDNN from your githab? Or can I just insert my
"cudnn-windows-x86_64_cuda12-archive" (which contains three more folders: 1bin 2inklude 3lib) following the instructions from your githab? Or leave it as it is, because I already have cuDNN installed.  As far as I understand they are the same programs or whatever they are called.

**Comment by bmaltais** [comment]
Good question. I don't own a 4080 so I can't test if the provided cudaNN files work with it. I know they work for 3090 and 4090... But not sure for other models.

It is not necessary to install the updated files. It make things slightly faster on 3090 and 4090 but it is not mandatory.
